scripts/globals/gear_sets.lua

```diff
@@ -256,6 +258,18 @@ function HandleHipsterSet(player, gearset, matches)
         -- printf("we have a special snowflake | gearset: %u | mod %u %u", gearset.id, MOD_FASTCAST, modValue);
         player:addGearSetMod(gearset.id, MOD_FASTCAST, modValue);
         return;
+    elseif (gearset.id == 113) then
+        local modValues = {0, 2, 5, 10, 15};
+        --  Phorcys Armor Set: Set Increases STR, VIT, INT, MND. Bonus is +2 Stats for 2 pieces, +5 for 3 pieces, +10 for 4 pieces, and +15 for 5 pieces
+        -- printf("we have a special snowflake | gearset: %u | mod %u %u", gearset.id, MOD_STR, modValues);
+        -- printf("we have a special snowflake | gearset: %u | mod %u %u", gearset.id, MOD_VIT, modValues);
+        -- printf("we have a special snowflake | gearset: %u | mod %u %u", gearset.id, MOD_INT, modValues);
+        -- printf("we have a special snowflake | gearset: %u | mod %u %u", gearset.id, MOD_MND, modValues);
+        player:addGearSetMod(gearset.id, MOD_STR, modValues[matches]);
+        player:addGearSetMod(gearset.id, MOD_VIT, modValues[matches]);
+        player:addGearSetMod(gearset.id, MOD_INT, modValues[matches]);
+        player:addGearSetMod(gearset.id, MOD_MND, modValues[matches]);
```

Comment:
@ibm2431 Thanks for the suggestion. I think I see what you mean. 1 piece, it will use first array value, so 0. 2, it will use the second array value, so 2. 3, it will use the third array value, so 5. Etc. Correct? I still need to retest, as my test using the if statements failed to add the values. In fact, even the matches = 5, value = 15, didn't work yesterday, so I assume there is something wrong in the value down here...